from spaceone.core.error import *


class ERROR_AWS_CONNECTION(ERROR_CONNECTOR):
    _message = '{reason}'
